android-tests/app/src/androidTest/java/com/azure/autoresttest/BodyStringSyncClientTests.java

```diff
@@ -0,0 +1,135 @@
+
+    @Test
+    public void getMbcs() throws Exception {
+        String result = client.getMbcsWithRestResponse().getValue();
+        String expected = "啊齄丂狛狜隣郎隣兀﨩ˊ〞〡￤℡㈱‐ー﹡﹢﹫、〓ⅰⅹ⒈€㈠㈩ⅠⅫ！￣ぁんァヶΑ︴АЯаяāɡㄅㄩ─╋︵﹄︻︱︳︴ⅰⅹɑ\uE7C7ɡ〇〾⿻⺁\uE843䜣\uE864€";
+        Assert.assertEquals(expected, result);
+    }
+
+    @Test
+    public void putMbcs() throws Exception {
+        Response<Void> putResult = client.putMbcsWithRestResponse();
+        assertEquals(200, putResult.getStatusCode());
+    }
+
+    @Test
+    public void getWhitespace() throws Exception {
+        String result = client.getWhitespaceWithRestResponse().getValue();
+        Assert.assertEquals("    Now is the time for all good men to come to the aid of their country    ", result);
+    }
+
+    @Test
+    public void putWhitespace() throws Exception {
+        Response<Void> putResult = client.putWhitespaceWithRestResponse();
+        assertEquals(200, putResult.getStatusCode());
+    }
+
+    @Test
+    public void getNotProvided() throws Exception {
+        try {
+            client.getNotProvidedWithRestResponse();
+        } catch (Exception ex) {
+            Assert.assertEquals(HttpResponseException.class, ex.getClass());
+            assertTrue(ex.getMessage().contains("JsonMappingException"));
+        }
+    }
+
+    @Test
+    public void getBase64Encoded() throws Exception {
+        byte[] result = client.getBase64EncodedWithRestResponse().getValue();
+        Assert.assertEquals("a string that gets encoded with base64", new String(result, StandardCharsets.UTF_8));
+    }
+
+    // copied from azure-core
+    private static String unquote(String string) {
+        if (string != null && !string.isEmpty()) {
+            final char firstCharacter = string.charAt(0);
+            if (firstCharacter == '\"' || firstCharacter == '\'') {
+                final int base64UrlStringLength = string.length();
+                final char lastCharacter = string.charAt(base64UrlStringLength - 1);
+                if (lastCharacter == firstCharacter) {
+                    string = string.substring(1, base64UrlStringLength - 1);
+                }
+            }
+        }
+        return string;
+    }
+
+    @Test
+    public void getBase64UrlEncoded() throws Exception {
+        byte[] result = client.getBase64EncodedWithRestResponse().getValue();
+        String resultString = new String(result);
+        Assert.assertEquals("a string that gets encoded with base64", resultString);
+    }
+
+    @Test
+    public void getNullBase64UrlEncoded() throws Exception {
+        Base64Url result = client.getNullBase64UrlEncodedWithRestResponse().getValue();
+        Assert.assertNull(result);
+    }
+
+    @Test
+    public void putBase64UrlEncoded() throws Exception {
+        try {
+            Response<Void> putResult = client.putBase64UrlEncodedWithRestResponse(new Base64Url("http://myhost.com/path?email=a.b@c.com&value=a%20B"));
+            assertEquals(200, putResult.getStatusCode());
```

Comment:
the same question as above, we're expecting this to succeed, so try block may not be needed.